app/scripts/components/common/mapbox/layers/raster-timeseries.tsx

```diff
@@ -273,13 +273,34 @@ export function MapLayerRasterTimeseries(props: MapLayerRasterTimeseriesProps) {
         LOG && console.groupEnd();
         /* eslint-enable no-console */
 
-        const responseData = await requestQuickCache({
-          url: `${tileApiEndpointToUse}/mosaic/register`,
-          payload,
-          controller
-        });
-
-        setMosaicUrl(responseData.links[1].href);
+        let responseData;
+
+        try {
+          responseData = await requestQuickCache({
+            url: `${tileApiEndpointToUse}/searches/register`,
+            payload,
+            controller
+          });
+          const mosaicUrl = responseData.links[1].href;
+          setMosaicUrl(mosaicUrl.replace('/{tileMatrixSetId}', '/WebMercatorQuad'));
```

Comment:
probably nice to have tileMatrixSetId as a variable but this change works with the upgraded titiler-pgstac version ✅